### DISCUSSION TITLE
Discussion on whether this fork is illegal or not / How the MIT license works

### DISCUSSION THREAD

**Discussion by ghost** [faker-js/faker#215] [General]
The open source doesn't mean copyright-free. According to LICENSE file, this project is copyrighted under MIT. The copyright owner is Marak Squires. 

If you intend to continue ignoring this warning, we must take action on them.
We will send a DMCA takedown notice to github, Then take a another legal action
https://github.com/faker-js/faker/issues/213

**Comment by ghost** [comment]
For everyone: Do not use this library. It's completely illegal. You should pay some money for it.

**Reply by damienwebdev** [reply]
@wowowooooo please clarify your complaint viewed against the original License terms:

> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

To be clear on exactly what are we doing in this project:

1. We made a copy
2. We are modifying our copy
3. We are distributing it our copy as well as the original.

Subject the to the condition that we maintain a copy of the original License, which we have.

We have upheld the terms of this contract.

**Reply by javaguru** [reply]
Hi @damienwebdev 

Ok you respect the license but this forked project loses the spirit of his conceptor.. 

Was it the deal for @Marak to regain his account back on Github ? 
(@Marak closed the original project) 

Just a legitim question for me.. and many others coders.

**Reply by damienwebdev** [reply]
@javaguru what Marak wants to do with his codebase and career is entirely his right.

Similarly, what I elect to do with my fork is entirely my right, assuming I abide by the original license terms (which this project does).

At the end of Marak's fork, there were ~250 contributors and about 1500 commits.

This means that, at the very least, ~16% of "Marak's project" wasn't Marak's.

He decided to throw away all of their effort without asking them (examples can be provided, simply ask @Moosh-be )

We disagreed with that decision.

Anything beyond these considerations is moot.

**Comment by Shinigami92** [comment]
I assume you have read https://github.com/faker-js/faker/discussions/52#discussioncomment-1955832

It is not illegal and even forking is a common standard in open source.

The license is also still valid as before and just taken over, see https://github.com/faker-js/faker/blob/a973ee1ce64d2b593f64b633e927382280da8f27/LICENSE#L26-L39

If you really think this repo/fork is illegal, so please also report every other fork that was created after 04. January 2022.
Good luck with that :shrug:.
Spoiler: You can invest your time in better stuff than this.

**Comment by fzn0x** [comment]
> All Open Source software can be used for commercial purpose; especially forked and modified.

**Comment by ghost** [comment]
Please note that Marak Squires holds the copyright of this project. Under the rules of contract law, Marak might be possible to revoke your license.

As well, the trademark is not under the MIT-license. You are not allowed to continue using trademark "faker.js".

So please stop hosting this repository immediately.

**Comment by prisis** [comment]
Good that we dont call this faker.js.

Please read [https://en.wikipedia.org/wiki/MIT_License](https://en.wikipedia.org/wiki/MIT_License) to understand what a MIT license is and how it can be used.

Spoiler: https://tmsearch.uspto.gov/bin/showfield?f=toc&state=4804%3A4837d9.1.1&p_search=searchss&p_L=50&BackReference=&p_plural=yes&p_s_PARA1=&p_tagrepl%7E%3A=PARA1%24LD&expr=PARA1+AND+PARA2&p_s_PARA2=+faker.js&p_tagrepl%7E%3A=PARA2%24COMB&p_op_ALL=AND&a_default=search&a_search=Submit+Query&a_search=Submit+Query faker.js is not registerd as a trademark.

**Comment by poyea** [comment]
Because the copyright notice is included [here](https://github.com/faker-js/faker/blob/main/LICENSE), it permits what people are working on here, including any modification, or even "selling" in the sense of receiving sponsorship. Despite this, actually, I'd love to see what GitHub's actions will be, if any.

**Comment by poyea** [comment]
I'd suggest we add/modify a paragraph on https://fakerjs.dev/update.html, so that a static reference is provided to avoid any copyright-related dispute.

**Reply by JessicaSachs** [reply]
(Being completely serious here, no snark: PRs welcome. Not sure what you’d expect to see written up.)

**Comment by rostislav-simonik** [comment]
Let's abandon any discussion about Marak, it's not worth of time. He is little sprat who doesn't know how to behave. He violated anything good that is on OSS. If he didn't see light at the end of the tunnel, he should just write on readme if somebody else can take over, and then to transfer ownership to given maintainers. Instead, he has taken damage-causing actions, because his business plan didn't go as he expected. I'm wondering if anybody would consider hiring anymore, with such reference he can start planting trees in New Zealand.

**Comment by wsdjeg** [comment]
Deleting source code is very wrong, but so is violating copyright!
